cypress/support/index.js

```diff
@@ -14,8 +14,53 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
+import {
+    applicationinventory,
+    button,
+    deleteAction,
+    tdTag,
+    trTag,
+} from "../integration/types/constants";
+import { actionButton } from "../integration/views/applicationinventory.view";
+import { navMenu } from "../integration/views/menu.view";
+import {
+    click,
+    clickByText,
+    deleteApplicationTableRows,
+    login,
+    logout,
+    selectItemsPerPage,
+} from "../utils/utils";
+import * as commonView from "../integration/views/common.view";
 import "./commands";
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 require("cypress-xpath"); // Refer - https://www.npmjs.com/package/cypress-xpath
+
+before("Login and Clean up Application Inventory Table Data", function () {
+    // Login
+    login();
+    cy.wait(2000);
+
+    // Navigate to application inventory tab
+    clickByText(navMenu, applicationinventory);
+    cy.wait(2000);
+
+    // Select 100 items per page
+    selectItemsPerPage(100);
+    cy.wait(2000);
+
+    // Check if the application inventory table is empty, else delete the existing rows
+    cy.get(commonView.appTable)
+        .next()
+        .then(($div) => {
+            if (!$div.hasClass("pf-c-empty-state")) {
+                // Delete all items of page
+                deleteApplicationTableRows();
+            }
+        });
+
+    // Logout
+    logout();
+});
```

Comment:
- Discussed with Nitish. As we want to clear applications created in application inventory before running the tests execution. 
- We found that this code of section gets executed before every spec which is not good while executing all the specs at a time. Hence keeping this PR in progress to research more about this section

Reply:
This will run before every spec. There is not suite level clean up available. Hence, removed this piece of code and using the clean up code specifically for import tests, where clean up is pre-requisite.